sass/_components.scss

```diff
@@ -654,4 +654,532 @@
         padding: $baseline;
         background: #363e4e;
     }
+}
+
+// Product Hero - 3x1
+//
+// .default - Default Hero
+// .hide-cta - Hero without CTA Button
+//
+// markup:
+// <div class="product-hero product-hero--3x1 {$modifiers}">
+//      <div class="columns is-mobile">
+//          <div class="column">
+//              <h2><span>What the hell is real vision television?</span></h2>
+//          </div>
+//      </div>
+//      <div class="columns">
+//          <div class="column product-hero__cell-container">
+//              <div class="product-hero__cell">
+//                  <h3><span>Unbiased</span></h3>
+//                  <p><span>Financial Insight</span></p>
+//              </div>
+//              <div class="product-hero__cell">
+//                  <h3><span>Unfiltered</span></h3>
+//                  <p><span>Actionable Ideas</span></p>
+//              </div>
+//              <div class="product-hero__cell">
+//                  <h3><span>Unapologetic</span></h3>
+//                  <p><span>Smart Opinions</span></p>
+//              </div>
+//          </div>
+//          <div class="column has-text-centered hero-cta__container">
+//              <a href="#" class="button">Read More</a>
+//          </div>
+//      </div>
+// </div>
+// 
+// Styleguide 5.11
+.product-hero {
+    &.product-hero--3x1 {
+        background: url('#{$root}/images/components/product-hero-bg.png');
+
+        h2 {
+            margin-bottom: $baseline;
+        }
+
+        .product-hero__cell-container {
+            padding-bottom: $baseline / 2;
+
+            @include mobile {
```

Comment:
Could you reverse this so mobile is the default and this is @include tablet (and the other occurences in this file)? I know we arent really mobile first, but it keeps things consistent with the other components.